src/pipeline_manager.py

```diff
@@ -6,26 +6,20 @@
 import json
 from pycocotools.coco import COCO
 
-from .pipeline_config import SOLUTION_CONFIG, CATEGORY_IDS, SEED, CATEGORY_LAYERS, ID_COLUMN
+from .pipeline_config import SOLUTION_CONFIG, SEED, ID_COLUMN
 from .pipelines import PIPELINES
 from .utils import init_logger, read_params, set_seed, get_img_ids_from_folder, map_evaluation, create_annotations, \
     generate_data_frame_chunks, NeptuneContext
 
-neptune_ctx = NeptuneContext()
-params = neptune_ctx.params
-ctx = neptune_ctx.ctx
-
-set_seed(SEED)
```

Comment:
@taraspiotr Why we get rid of `set_seed`?

Reply:
@apyskir We run `set_seed` in `PipelineManager` constructor